app/controllers/registrations_controller.rb

```diff
@@ -6,13 +6,14 @@ def new
   end
 
   def create
-    @user = User.new(user_params)
+    @user = User.new(user_params.except(:invite_code))
```

Comment:
I think you can remove this line, as it does nothing what AR will not do anyway. AR will ignore this param by default.